cloudinfo/catalog.go

```diff
@@ -0,0 +1,32 @@
+package cloudinfo
+
+import (
+	"fmt"
+	"github.com/IBM/platform-services-go-sdk/catalogmanagementv1"
+)
+
+func (infoSvc *CloudInfoService) GetVersion(versionLocator string) (*catalogmanagementv1.Version, error) {
```

Comment:
Since this function is public for the entire package cloudinfo, you might want to give it a more unique name like `GetCatalogVersionByLocator`